etcd-manager/pkg/etcd/etcdprocess.go

```diff
@@ -278,6 +279,17 @@ func (p *etcdProcess) Start() error {
 		klog.Warningf("using insecure configuration for etcd clients")
 	}
 
+	// etcd 3.5 had some corruption issues and recommends this setting,
+	// which is planned as the default in 3.6
+	version, err := semver.ParseTolerant(p.EtcdVersion)
+	if err != nil {
+		klog.Warningf("error parsing version %q: %v", p.EtcdVersion, err)
+	}
```

Comment:
Nit: In the past, this used to be done with just `HasPrefix`, probably to avoid the error handling, without exiting.
https://github.com/kubernetes-sigs/etcdadm/blob/11636446c1b71bde18a055180de1099b886f08d3/etcd-manager/pkg/etcd/etcdprocess.go#L241-L243

Reply:
Ah good point ... do you think it's a blocker?  I started the release process...

Reply:
Surely not a blocker :)